feeds/cache.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Cache
+parent: Feeds
+nav_order: 3
+---
+
+# Central Feed Cache
+
+Cache handling for feeds specified by `<PackageSource>` in Central Feed Management is separate from feeds specified in NuGet.config.
```

Comment:
Will it be part of `dotnet nuget locals --list all` ? so that the `--clear` will also apply
eg:
`dotnet nuget locals --clear contoso`